cbv3_django_prototype/config/settings/base.py

```diff
@@ -42,9 +42,21 @@
 # ------------------------------------------------------------------------------
 # https://docs.djangoproject.com/en/dev/ref/settings/#databases
 
+#DATABASES = {
+#    "default": env.db("DATABASE_URL", default="postgres:///cbv3_django_prototype")
+#}
+
 DATABASES = {
-    "default": env.db("DATABASE_URL", default="postgres:///cbv3_django_prototype")
+    'default': {
```

Comment:
Lets add instructions to the README about setting up the Postgres Docker container for this to work.

Reply:
Thanks for the suggestion. I remember this was a pain (IIRC Bethany actually helped me through this during the first/only hangout we've done for this repo at https://zoom.us/recording/play/Ttk6GpLLMJunjTjWpH577Q66cmnvs1dTGf9OCo2CzdwX45NGFFSUfzoIumAnMV0p?autoplay=true&startTime=1567053828000), so will file an issue/make a PR for it.